tests/specs/example.py

```diff
@@ -0,0 +1,44 @@
+from tests.fixtures import dims, facts
+
+TABLES = [dims.customer, dims.product, facts.transactions]
+
+SCENARIO = 'sqls/example.sql'
+
+SPECS = dict(
+    run_customers=dict(
+        scenario=SCENARIO,
+        description="""
+            Given 2 customers, one from UK and one from US
+            When selecting customers
+            Only customers from UK are selected
+        """,
+        tables=TABLES,
```

Comment:
It would be perfect from unit test perspective if also upper CTE's can be mocked, otherwise the deeper you go the closer your tests are to integration tests.

Reply:
I haven't thought about this. Good idea!

Reply:
Issue was created